build_tools/cmake/build_benchmarks.sh

```diff
@@ -35,6 +35,7 @@ echo "Configuring to build benchmarks"
   -DIREE_HOST_BINARY_ROOT="${IREE_HOST_BINARY_ROOT}" \
   -DIREE_BUILD_BENCHMARKS=ON \
   -DIREE_BUILD_MICROBENCHMARKS=ON \
+  -DIREE_BUILD_EXPERIMENTAL_PYTHON_GENERATED_BENCHMARKS=ON \
```

Comment:
Ah, I was thinking of a *separate* job that builds the new benchmark suites. I think this job is in the critical path for the CI latency, so it would be preferable to not make it longer

Reply:
I'm okay with creating a separate job. But this actually only increases the step by 30 seconds (among of 11m30s, due to the core counts and low parallelism of this step).

It might be actually slower if we create a separate job because of the overhead of setup steps.

Reply:
Oh nevermind then